app/templates/preprints/provider/preprint-detail.hbs

```diff
@@ -136,4 +136,17 @@
 
         </div>
     </div>
+
+    {{#if showWarning}}
+        {{#bs-modal onHide=(action 'hideWarning') as |modal|}}
+            {{#modal.header}}
+                <div class="warning-header"><i class="fa fa-exclamation-triangle"></i> {{t 'content.warning.header'}}</div>
+            {{/modal.header}}
+            {{#modal.body}}<div class="warning-body">{{t 'content.warning.body'}}</div>{{/modal.body}}
+            {{#modal.footer}}
+                {{#bs-button onClick=(action 'leavePage')}}{{t 'content.warning.footer.leave'}}{{/bs-button}}
+                {{#bs-button onClick=(action 'hideWarning') type="info"}}{{t 'content.warning.footer.stay'}}{{/bs-button}}
```

Comment:
These buttons are switched compared to the buttons in the browser modal:
<img width="289" alt="screen shot 2017-12-15 at 2 03 54 pm" src="https://user-images.githubusercontent.com/6776190/34057307-67c85f7e-e1a4-11e7-8993-113957d8e9d8.png">
<img width="199" alt="screen shot 2017-12-15 at 2 03 39 pm" src="https://user-images.githubusercontent.com/6776190/34057343-8b7a2434-e1a4-11e7-82d0-1cada643c96f.png">
<img width="262" alt="screen shot 2017-12-15 at 2 29 59 pm" src="https://user-images.githubusercontent.com/6776190/34057338-86576c14-e1a4-11e7-8cf7-040606d37dda.png">
<img width="192" alt="screen shot 2017-12-15 at 2 30 10 pm" src="https://user-images.githubusercontent.com/6776190/34057347-8f24222e-e1a4-11e7-9002-a6915c591432.png">

Reply:
In the ticket --> developer notes, the "leave page" is on the left. I agree the order should be consistent with the browser modal, will update.